packages/doc/src/components/CodeBlock/CodeBlock.jsx

```diff
@@ -176,10 +175,10 @@ const CodeBlock = ({ children, reactLive, center }) => {
 
   return reactLive ? (
     <MDXContext.Consumer>
-      {scope => (
+      {Scope => (
         <LiveProvider
```

Comment:
We changed 'Scope' to capital so we could test the LivePreview
I guess we can change back to lowercase

Reply:
resolved!